mcp/pacemaker.c

```diff
@@ -913,6 +914,8 @@ main(int argc, char **argv)
     int option_index = 0;
     gboolean shutdown = FALSE;
 
+    gboolean start_standby = FALSE;
```

Comment:
It's perfectly fine to do it this way, but just to give a couple of alternatives for discussion, if you used:

    const char *start_standby = "default";

and similarly in the 's' case, then you would only need one setenv() line instead of the block below.

And since not setting the variable is equivalent to the default, an even simpler alternative would be to call setenv() directly in the 's' case, and not do anything elsewhere.

When comparing equally sufficient alternatives, simpler is usually better, but even more important is readability (whichever alternative is easiest to comprehend when reading the code). But this is more a matter of personal preference than anything important.